src/merge-pr.js

```diff
@@ -0,0 +1,62 @@
+
+var flagutil = require('./flagutil');
+var optimist = require('optimist');
+var shelljs = require('shelljs');
+var executil = require('./executil');
+var gitutil = require('./gitutil');
+var superspawn = require('./superspawn');
+
+module.exports = function *(argv) {
+    var opt = flagutil.registerHelpFlag(optimist);
+    opt.options('pr', {
+            desc: 'PR # that needs to be merged',
+            demand: true
+        }).options('remote', {
```

Comment:
I know that would be nice - might get to it in the next iteration. Out of time for this one now.